app/assets/stylesheets/forever_style_guide/globals/_effects.scss

```diff
@@ -18,6 +18,12 @@
   transition: $transition-properties;
 }
 
+@mixin transform($transform-properties...) {
```

Comment:
Should we just add the autoprefixer gem to the styleguide? 
https://rubygems.org/gems/autoprefixer-rails/versions/6.3.6.1
https://github.com/ai/autoprefixer-rails